spring-boot-project/spring-boot-test-autoconfigure/src/main/java/org/springframework/boot/test/autoconfigure/jdbc/AutoConfigureTestDatabase.java

```diff
@@ -60,6 +60,15 @@
 	 */
 	EmbeddedDatabaseConnection connection() default EmbeddedDatabaseConnection.NONE;
 
+	/**
+	 * The custom connection URL to use. If specified it will override the default URL
+	 * string in the embedded datasource. It is expected to be fully specified and no
```

Comment:
I am not sure this brings us much to mention that. One might argue that using its own URL means that another feature is lost (unique names) so perhaps we should do something about it.